.packaging/rust/src/bin/main.rs

```diff
@@ -0,0 +1,39 @@
+use enzyme::Repo::{self, *};
+fn main() {
+    match get_enzyme() {
+        Ok(()) => {}
+        Err(e) => panic!("building failed: {}", e),
+    }
+}
+
+fn get_enzyme() -> Result<(), String> {
+    let mut cmd_line = std::env::args();
+    if cmd_line.len() == 2 {
+        let arg = cmd_line
+            .nth(1)
+            .expect("failed reading")
+            .parse::<Repo>()
+            .expect("failed reading first argument!");
+
+        enzyme::download(arg.clone())?;
```

Comment:
there should probably be an option to just get enzyme from ../../../../enzyme.

Reply:
+1

Reply:
Added.